packages/launch/src/focused-launch/index.tsx

```diff
@@ -46,11 +46,30 @@ const FocusedLaunch: React.FunctionComponent = () => {
 	const planProductIdFromCart = usePlanProductIdFromCart();
 	const { setPlanProductId } = useDispatch( LAUNCH_STORE );
 
+	// Once there is a selected plan, do not retrieve plan from from cart anymore.
+	const canUsePlanProductIdFromCart = React.useRef( true );
 	React.useEffect( () => {
-		if ( ! selectedPlanProductId && planProductIdFromCart && ! hasPaidPlan ) {
+		if ( selectedPlanProductId ) {
+			canUsePlanProductIdFromCart.current = false;
+		}
+	}, [ selectedPlanProductId ] );
+
+	React.useEffect( () => {
+		if (
+			! selectedPlanProductId &&
+			planProductIdFromCart &&
+			! hasPaidPlan &&
+			canUsePlanProductIdFromCart.current
```

Comment:
Set `setPlanProductId` when:
* I haven't selected a plan and;
* The checkout cart contains a paid plan and;
* haven't purchased a paid plan already and;
* It's okay to select a plan from the cart **.

** `useRef` is persistent through render cycles, hence it makes sense to add this. Without it, it's still possible to select the store's plan causing an ugly-ish jump while selecting a custom domain.

Reply:
Thanks for the summary! <3